src/i18n/locale/es.json

```diff
@@ -299,7 +299,8 @@
                 "static_navigation": "Navigación Estática",
                 "show_page_number": "Mostrar numero de pagina",
                 "reader_type": "Tipo de Lector",
-                "skip_dup_chapters": "Omitir capítulos duplicados"
+                "skip_dup_chapters": "Omitir capítulos duplicados",
+                "fit_to_window": "Ajustar página a la ventana"
```

Comment:
nitpick, but you've forgot the translation key 😄

Reply:
good catch